src/renderer/components/+namespaces/namespace.store.ts

```diff
@@ -5,7 +5,7 @@ import { Namespace, namespacesApi } from "../../api/endpoints/namespaces.api";
 import { createPageParam } from "../../navigation";
 import { apiManager } from "../../api/api-manager";
 
-const selectedNamespaces = createStorage<string[]>("selected_namespaces");
+const selectedNamespaces = createStorage<string[]>("selected_namespaces", []);
```

Comment:
Default value here must be `undefined` since it breaks logic on first app openning:

```ts
    const prevSelectedNamespaces = selectedNamespaces.get();

    // return previously saved namespaces from local-storage (if any)
    if (prevSelectedNamespaces) {
      return prevSelectedNamespaces.filter(namespace => namespaces.has(namespace));
    }
```

Reply:
Ah thanks, will rework this.